Sources/OktaIdxAuth/OktaIdxAuth.swift

```diff
@@ -91,6 +92,24 @@ import OktaIdx
         }
     }
     
+    @objc(OktaIdxAuthSocialAuthentication)
+    public class SocialAuth: NSObject {
```

Comment:
Is there a reason you created a nested class for this? Are you planning for future expansion, or was it to just add name spacing to the class?

Since the word "Auth" already exists in the class name (e.g. OktaIdxAuth), I'd suggest we simplify the class name a bit. I'm thinking maybe `OktaIdxAuthSocialOptions` and have the name in Swift be `OktaIdxAuth.SocialOptions`.

Reply:
I would like to separate SocialAuth objects, so yes it's mostly for name spacing reasons. Also, I had additional class here, but no needs now. So refactored that.